app/store/bot.data.ts

```diff
@@ -113,7 +117,7 @@ export const createEmptyBot = (): Bot => ({
   createdAt: Date.now(),
   botHello: Locale.Store.BotHello,
   session: createEmptySession(),
-  datasource: "documents",
+  datasource: "",
```

Comment:
Why not use documents as default?

Reply:
how about using `LLAMA_CLOUD_INDEX_NAME`? the problem with it being undefined is that a user can create a new bot without editing it and then he would chat with an undefined data source.

Reply:
yes

Reply:
this would prevent this issue: "Yes, in that case, the datasource will be undefined, and I am throwing an error message when the datasource is missing in the /chat API.""